packages/simplebar/src/scrollbar-width.js

```diff
@@ -1,12 +1,14 @@
 let cachedScrollbarWidth = null;
 let cachedDevicePixelRatio = null;
 
-window.addEventListener('resize', () => {
-  if (cachedDevicePixelRatio !== window.devicePixelRatio) {
-    cachedDevicePixelRatio = window.devicePixelRatio;
-    cachedScrollbarWidth = null;
-  }
-});
+if (typeof window !== 'undefined') {
+  window.addEventListener('resize', () => {
```

Comment:
Could you use `canUseDOM` like done [here](https://github.com/Grsmto/simplebar/blob/master/packages/simplebar/src/simplebar.js#L150) instead of hardcoded `typeof window` test? So we keep consistency.
Thanks!